src/main/java/dog/pawbook/model/owner/Owner.java

```diff
@@ -92,6 +99,7 @@ public boolean equals(Object other) {
                 && otherOwner.getPhone().equals(getPhone())
                 && otherOwner.getEmail().equals(getEmail())
                 && otherOwner.getAddress().equals(getAddress())
+                && String.valueOf(otherOwner.getID()).equals(String.valueOf(getID()))
```

Comment:
isn't it just `otherOwner.getID() == getID()`? and I think we can do away with this check since it is reasonable enough that 2 owners with all details that are the same point to the same person, and some tests also uses this method to compare between owners.

Reply:
Thanks for pointing this out! Will amend the code now!